airbyte-integrations/connectors/source-retently/source_retently/source.py

```diff
@@ -80,6 +80,14 @@ def parse_response(
         stream_data = data.get(self.json_path) if self.json_path else data
         yield from stream_data
 
+    @staticmethod
+    def convert_empty_string_to_null(record: MutableMapping[str, Any], parent_key: str, field_key: str):
+        """
+        Converts empty strings to null in the specified field of the record.
+        """
+        if parent_key in record and field_key in record[parent_key] and record[parent_key][field_key] == "":
+            record[parent_key][field_key] = None
```

Comment:
The current implementation is not secure. It will break in the `record[parent_key][field_key]` step.
```suggestion
        if record.get(parent_key, {}).get(field_key, "") == "":
            record[parent_key][field_key] = None
```
Is safe and won't break in runtime.